pandas/core/arrays/masked.py

```diff
@@ -1492,6 +1492,9 @@ def _groupby_op(
         else:
             result_mask = np.zeros(ngroups, dtype=bool)
 
+        if op.how == "rank" and kwargs.get("na_option") in ["top", "bottom"]:
```

Comment:
nit: can just use `how` here instead of `op.how`. It avoids an attribute access, and relies less on the state of WrappedCythonOp.